ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/port-channel-interfaces.j2

```diff
@@ -6,8 +6,8 @@
 
 #### L2
 
-| Interface | Description | Type | Mode | VLANs | Native VLAN | Trunk Group | LACP Fallback Timeout | LACP Fallback Mode | MLAG ID | EVPN ESI |
-| --------- | ----------- | ---- | ---- | ----- | ----------- | ------------| --------------------- | ------------------ | ------- | -------- |
+| Interface | Description | Type | Mode | VLANs | Native VLAN | Trunk Group | LACP Fallback Timeout | LACP Fallback Mode | MLAG ID | EVPN ESI | Multihoming Redundancy Mode | Designated Forwarder Election Algorithm | Preference Value | Dont Preempt |
```

Comment:
1. We should avoid adding this long headings on general tables.
2. Adding these to the general L2 table will render these extra fields on *all* deployments for *any* device with port_channels, even if they don't run EVPN.
I think a separate `EVPN Multihoming` table is better, which is only shown if `esi` key is defined under each port_channel.

Reply:
Sure, i'll add a new table something similar to

#### EVPN Multihoming

| EVPN ESI | Multihoming Redundancy Mode | Designated Forwarder Election Algorithm | Preference Value | Dont Preempt |
| -------- | --------------------------- | --------------------------------------- | ---------------- | ------------ |
| 0000:0000:0101:0102:0033 | single-active | preference | 1000 | True |
| 0000:0000:0101:0102:0044 | - | modulus | - | - |
| 0000:0000:0101:0102:0033 | all-active | preference | 2000 | False |

Reply:
or do we keep the EVPN ESI part of L2 port-channel table, and just add the new fields under the new `EVPN Multihoming` table?